### PR TITLE
migration-RollupProductDevDoc.tsx-to-tailwind

### PR DESCRIPTION
**Pull request by cqw-acq**
migration-RollupProductDevDoc.tsx-to-tailwind
## Description

i migrationed  `RollupProductDevDoc.tsx` to tailwind

**### Snapshot**

localhost:
截屏2024-10-31 11 13 00

ethereum.org:
截屏2024-10-31 11 13 05

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/13946

### CODE REVIEW

File: src/components/RollupProductDevDoc.tsx [hunk 1]
```diff
@@ -14,51 +11,45 @@ export type RollupProductDevDocProps = {
...
-              <Box p={4} pb={0}>
-                <Box>
-                  <Heading
-                    as="h4"
-                    fontSize={{ base: "md", md: "xl" }}
-                    fontWeight="500"
-                    lineHeight="1.4"
-                    my={4}
-                  >
+            <div key={idx} className="bg-rollupDevDocList my-4">
```
Comment (pettinarip) [inline src/components/RollupProductDevDoc.tsx:18]:
```suggestion
            <div key={idx} className="bg-background-highlight my-4">
```

File: src/components/RollupProductDevDoc.tsx [hunk 2]
```diff
@@ -14,51 +11,45 @@ export type RollupProductDevDocProps = {
...
                   {noteKey.length > 0 && (
-                    <Text>
+                    <p>
                       * <Translation id={`page-layer-2:${noteKey}`} />
-                    </Text>
+                    </p>
                   )}
-                  <UnorderedList>
-                    <ListItem>
+                  <ul className="list-disc pl-5">
```
Comment (pettinarip) [inline src/components/RollupProductDevDoc.tsx:29]: Could we use to the shadcn components under https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/list.tsx ?

You should use the exported `UnorderedList` and `ListItem` in this case.

Comment (pettinarip) [inline src/components/RollupProductDevDoc.tsx:29]:
```suggestion
                  <ul className="list-disc">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
@cqw-acq thanks for the PR! almost there, left a few tiny changes.

File: src/components/RollupProductDevDoc.tsx (hunk 2 above)
Comment (cqw-acq) [reply src/components/RollupProductDevDoc.tsx:29]: ok，i will do it now

Comment (cqw-acq) [reply src/components/RollupProductDevDoc.tsx:29]: so i need to make it back?

Comment (cqw-acq) [reply src/components/RollupProductDevDoc.tsx]: done

**Comment by cqw-acq** [comment]
@pettinarip done!

File: src/components/RollupProductDevDoc.tsx [hunk 3]
```diff
@@ -14,26 +13,20 @@ export type RollupProductDevDocProps = {
...
+            <div key={idx} className="my-4 bg-background-highlight">
+              <div className="p-4 pb-0">
+                <div>
+                  <h4 className="my-4 text-md font-medium leading-relaxed md:text-xl">
                     {name}
-                  </Heading>
+                  </h4>
                   {noteKey.length > 0 && (
-                    <Text>
+                    <p>
```
Comment (pettinarip) [inline src/components/RollupProductDevDoc.tsx:27]:
```suggestion
                    <p className="mb-4">
```

**Review by pettinarip — APPROVED** [review]
LGTM thanks @cqw-acq

**Pull request merged**